Bukkit/build.gradle

```diff
@@ -29,6 +30,7 @@ dependencies {
         exclude(module: "bukkit")
     }
     implementation("me.clip:placeholderapi:2.10.4")
+    compile("se.hyperver.hyperverse:Core:0.6.0-SNAPSHOT")
```

Comment:
I'd add `{ transitive = false }` because we don't have stuff like taskchain, acf or hocon on our maven and it's not needed.